lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -519,26 +524,35 @@ func (c *Client) sharedDirectoryInfoRequest(req tdp.SharedDirectoryInfoRequest)
 	return C.ErrCodeSuccess
 }
 
-// close frees the memory of the cgo.Handle,
-// closes the RDP client connection,
-// and frees the Rust client.
+// close closes the RDP client connection and
+// the TDP connection to the browser.
 func (c *Client) close() {
 	c.closeOnce.Do(func() {
 		// Ensure the RDP connection is closed
 		if errCode := C.close_rdp(c.rustClient); errCode != C.ErrCodeSuccess {
 			c.cfg.Log.Warningf("error closing the RDP connection")
 		}
+		c.cfg.Log.Debug(("RDP connection closed successfully"))
```

Comment:
- It wasn't necessarily closed successfully since the code falls through to this case on error as well.
- Are there some extra `((` `))` here?